src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs

```diff
@@ -1166,6 +1193,8 @@ protected override void UpdateImage()
             }
 
             base.UpdateImage();
+
+            // TODO : We should move InternalSavedDynamicPropertyCallbacks now, becaused visual was changed, but URL was not changed!
```

Comment:
typo

```suggestion
            // TODO : We should move InternalSavedDynamicPropertyCallbacks now because visual was changed, but URL was not changed!
```

or to make it more naturally :

```suggestion
            // TODO : It is necessary to relocate `InternalSavedDynamicPropertyCallbacks` as the visuals have been altered, while the URL remains unchanged
```

Reply:
Fixed done! The sentenses looks more clear then before :D